src/core/frontend/framebuffer_layout.cpp

```diff
@@ -443,8 +443,19 @@ FramebufferLayout FrameLayoutFromResolutionScale(u32 res_scale, bool is_secondar
                                     Settings::values.upright_screen.GetValue(), 1,
                                     Settings::SmallScreenPosition::MiddleRight);
 
-        case Settings::LayoutOption::Default:
+        case Settings::LayoutOption::HybridScreen:
         default:
```

Comment:
I'm not sure if the `default` case being up here was a mistake from the author that went unnoticed or if there was some reason behind this, but I can't think of a logical reason why it would be here.